plugin/ale.vim

```diff
@@ -186,6 +186,15 @@ let g:ale_deno_executable = get(g:, 'ale_deno_executable', 'deno')
 " If 1, enable a popup menu for commands.
 let g:ale_popup_menu_enabled = get(g:, 'ale_popup_menu_enabled', has('gui_running'))
 
+" If 1, disables ALE's built in error display. Instead, all errors are piped
+" to Neovim's diagnostics API.
+let g:ale_send_to_neovim_diagnostics = get(g:, 'ale_send_to_neovim_diagnostics', 0)
```

Comment:
We could name this `ale_send_diagnostics_to_neovim`. It might still sound a bit strange, because we're sending data via the NeoVim diagnostics API.

I think this variable name sound a bit odd, but I can't think of a better one, so don't change it.

Reply:
I think for future-proofing, renaming to `ale_use_diagnostics_api` is better in case Vim proper ever gains support (and not necessarily via Lua even).